internal/model/netx.go

```diff
@@ -303,6 +303,26 @@ type Trace interface {
 	// can use functionality exported by the ./internal/testingx pkg.
 	TimeNow() time.Time
 
+	// OnDNSLookupDone is called when DNSLookup terminates
+	//
+	// Arguments:
```

Comment:
Could you please document which arguments could be nil? (If other functions above do not do that, please apply the boy scout rule and ensure all other functions in this file document which arguments could be nil)

Reply:
A little improvement on the docs. Maybe this grasps what we expect in arguments better: 
https://github.com/ooni/probe-cli/pull/831/commits/05369a38b3a7a3d71e1f71c1b09a773ad477a228#diff-f9f42daa515ddf8adbc780a80668686e9e2425baff569b86c38e01c1f2449636

Reply:
Yeah!